entry_types/scrolled/package/src/frontend/ContentElement.module.css

```diff
@@ -79,5 +78,6 @@
 .selected:hover:after,
 .selected:after {
   border: solid 1px currentColor;
-  opacity: 0.8
+  opacity: 0.8;
+  pointer-events: none;
```

Comment:
This change makes the [specs fail](https://travis-ci.org/codevise/pageflow/jobs/651980712#L2795) since [the click](https://github.com/codevise/pageflow/blob/da1b7e220d4f71ddb2ca70d3f536e62be2450705/entry_types/scrolled/spec/features/entry_editor/editing_a_content_element_spec.rb#L19) on the content element is intercepted by the pseudo element.

Reply:
I saw that too.
The behavior is as desired, but I need to come up with an idea for a test-compliant change.